seed/models/data_quality.py

```diff
@@ -1007,6 +1052,8 @@ def add_invalid_geometry_entry_provided(self, row_id, rule, display_name, value)
             'message': display_name + ' should be in WKT format',
             'detailed_message': detailed_message,
             'severity': rule.get_severity_display(),
+            'condition': rule.condition,
+            'label': rule.status_label,
```

Comment:
As discussed separately, we'll be moving these potential fixes into another PR/Issue. There's likely other things to check, since I recall DQ labels being updated previously. It'd be worth deeper review/inspection of required changes for that bug fix.